packages/oidc-client/src/initWorker.ts

```diff
@@ -189,6 +189,8 @@ export const initWorkerAsync = async(serviceWorkerRelativeUrl, configurationName
 
     try {
         await navigator.serviceWorker.ready;
+        if (!navigator.serviceWorker.controller)
```

Comment:
Hi @VladimirPlatonenko ,

Thank you so much for your help!

Very sorry for the delay. I will try to test it as soon as i can.
The modification have to be done in the service worker packages.

Here, in this file, it won't affect the published library.

Reply:
I moved changes to ts file for service